content/reports/1000_IBM Power - Virtual Machines/010_All Power VMs.yaml

```diff
@@ -0,0 +1,87 @@
+---
+- MiqReport:
```

Comment:
I think it's better if I remove this line

Reply:
This is the format for a YAML array, you want to remove the `- MiqReport:` and drop all the leading indentation

Reply:
```
>> MiqReport.last
=> #<MiqReport id: 153, name: "PowerVC VMs", title: "IBM PowerVC VMs", rpt_group: "IBM Power - Virtual Machines", rpt_type: "Default", ...
```

Hi @agrare, Do you know what console I have to launch to type this?

Reply:
@CallisteH that is just `rails c`